src/internal/connector/exchange/restore_test.go

```diff
@@ -125,6 +125,41 @@ func (suite *ExchangeRestoreSuite) TestRestoreEvent() {
 		fault.New(true))
 	assert.NoError(t, err, clues.ToCore(err))
 	assert.NotNil(t, info, "event item info")
+
+	// Recurrence testing
+	// 1. Event doesn't have recurrence timezone specified
+	info, err = RestoreExchangeEvent(ctx,
+		mockconnector.GetMockEventWithRecurrenceBytes(name, false, ""),
+		suite.gs,
+		control.Copy,
+		calendarID,
+		userID,
+		fault.New(true))
+	assert.NoError(t, err, clues.ToCore(err))
+	assert.NotNil(t, info, "event item info")
```

Comment:
Having a non-nil item isn't very pertinent to this test case.  Could you add an assertion for the recurrence timezone, to ensure the restored object matches your new conditional?

Reply:
Good catch. Will do

Reply:
We won't be able to check recurrence timezone details in the integration test ( we only have access to details.ExchangeInfo).
I'll add a UT for ToEventSimplified for this purpose.